scripts/bump-spin-formula.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+VERSION=$1
+FORMULA=${2:-Formula/Spin.rb}
+
+usage() {
+    echo "Usage: $0 <VERSION> [<FORMULA_PATH>]"
+    echo "Updates the Spin Formula for the specified release version"
+    echo "Example: $0 3.0.0"
+}
+
+if [ $# -ne 1 ]; then
+    usage
+    exit 1
+fi
+
+# Get the checksum file for the release
+wget -qO checksums.txt "https://github.com/fermyon/spin/releases/download/$VERSION/checksums-$VERSION.txt"
```

Comment:
Can we error and exit here when the checksums.txt download is empty or otherwise invalid?  (eg when release does not exist, etc)